Add security-council doc and updated 2 docs (#75)
maticnetwork/zkevm-docs: docs/protocol/upgradability.md

  - openzeppelin-upgrades
---

It is inevitable that the current version of the Polygon zkEVM will go through updates and some upgrades, as it gets tested by both the community of Polygon developers and the internal team.

For this reason, and as an effort towards incentivising developers to battle-test the Polygon zkEVM, [bug-bounties](https://immunefi.com/bounty/polygonzkevm/) have been made available.

> Since zk-rollup systems are nascent, it is expected that the frequency of upgrades should decline with time.

Reviewer: ```suggestion
Since zk-Rollup ecosystems are nascent, it is expected that the frequency of upgrades should decline with time.
```

> In tandem, Polygon intends to move its governance of upgrades from the currently centralised approach to a much more decentralised modus operandi.

Reviewer: ```suggestion
In tandem, Polygon intends to move its governance of upgrades from the currently centralized approach to a much more decentralized modus operandi.
```

These gradual _changes in governance_ will follow Polygon Improvement Proposals (PIPs), as already outlined in Polygon's [Three Pillars of Governance](https://forum.polygon.technology/t/the-three-pillars-of-polygon-governance-call-for-proposals/11847).

> Presently, centralisation is seen in the form of the _Admin Multisig Contract_ and the [_Security Council Multisig_](/docs/protocol/security-council.md#security-council.md).

Reviewer: ```suggestion
Presently, centralization is seen in the form of the _Admin Multisig Contract_ and the [_Security Council Multisig_](/docs/protocol/security-council.md#security-council.md).
```
